createcourse/tasks.py

```diff
@@ -16,10 +14,16 @@
 from .api_settings import workshop_url, yaksh_post_url, workshop_json_file_mapper
 import os
 
+from django_celery_results.models import TaskResult
 
 @celery.task
 def fetch_data():
     params = {'status': 1}
+    last_fetched_task=TaskResult.objects.order_by('-date_done')
+    if len(last_fetched_task):
```

Comment:
Don't do it like this. `TaskResult.objects.latest()`  try this. Check the Django documentation about how to use the `latest`.